python-app/doc/action.yml

```diff
@@ -35,5 +35,5 @@ runs:
 
     - if: ${{ steps.check-doc.outputs.is-doc-needed == 'true' }}
       name: Publish to redocly
-      run: npx @redocly/openapi-cli@latest push doc/api-spec.yaml "${{ inputs.redocly-project }}"
+      run: npx @redocly/cli@latest push doc/api-spec.yaml "${{ inputs.redocly-project }}"
```

Comment:
just wondering, should this be using our own redocly action? If we do so, it will at least handle some errors properly, but I am not sure what else it brings.

Reply:
I asked myself the same question. I choose to submit the deprecation fix + my changes without touching anything else but I think it would be better to re-use our own so this action benefit from any improvement.
Our action has a bit of error management but the CLI is not very well done on error handling (like prompting for credentials if they are missing without any way to says "hey it's headless don't prompt, it will lock the workflow"). I hesitated to submit an issue on the redocly-cli repository

Reply:
Cool, let's do that in a second PR then 👍